### DISCUSSION TITLE
[A-L] (2023/24) Foglio 4 - Esercizio 8

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/calcolo-delle-probabilita#162] [Esercizi - Bertini] [answered]
![Screenshot 2023-10-23 185210](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/107944476/e0331b99-8f5c-4d33-bee9-67a28b0a7538)

**Comment by Elia-Belli** [comment, marked as answer]
### Interpretazione Probabilistica data in Classe
- **(1)** Consideriamo un'estrazione di $n$ palline senza rimpiazzo da una scatola che contiene $B$ palline bianche e $N$ palline nere. La probabilità di estrarre $k$ palline bianche:
$$\mathbb{P}(k\ delle\ n\ estratte\ sono\ Bianche\ )=\frac{\binom{B}{k}\binom{N}{n-k}}{\binom{B+N}{n}}$$
Adesso facciamo variare $k$ e sfruttiamo l'additività:
$$\sum^n_{k=0}\mathbb{P}(k\ delle\ n\ estratte\ sono\ Bianche\ )=1$$
### Soluzione Analoga a quella in Classe
- **(2)** Consideriamo lo schema di Bernoulli con $p=1/2$ che esca testa(1), e $1-p=1/2$ che esca croce(0).
Su $n$ lanci otteniamo esattamente $k$ teste con $\mathbb{P}(T_k)=\binom{n}{k}\cdot p^{k}\cdot (1-p)^{n-k}=\binom{n}{k}\cdot p^{n}$
Gli $n$ lanci di Alice sono indipendenti dagli $n$ lanci di Bob, quindi la probabilità che fissato il numero di teste $k$ ottengano entrambi $k$ teste è il prodotto delle probabilità che ognuno ne faccia $k$.
Detto questo è evidente che l'evento in cui entrambi fanno $0$ teste sia disgiunto da quello in cui ne fanno $1,2,3,..,n$ e quindi basta sommare per tutti i possibili valori di $k$ :
$$\mathbb{P}(A_T=B_T)=\sum^n_{k=0}\binom{n}{k}\cdot p^n\cdot \binom{n}{k}p^n=p^{2n}\sum^n_{k=0}\binom{n}{k}\binom{n}{k}$$
Siamo giunti ad un caso particolare della formula nel punto **(1)**, con $B=N=n$, ricordando che $\binom{n}{k}=\binom{n}{n-k}$:
$$\mathbb{P}(A_T=B_T)=p^{2n}\binom{2n}{n}=(\frac{1}{4})^n\binom{2n}{n}$$
**Osservazione**: la probabilità di ottenere lo stesso numero di teste diminuisce con l'aumentare dei lanci, infatti:
$$\lim_{n\rightarrow \infty}p^{2n}\frac{(2n)!}{(n!)^2}=0$$

**Comment by CiottoloMaggico** [comment]
Soluzione esagerata. (Aiuto del prof)
![image](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/79638284/d13b2dab-005d-48f0-8bbe-e0317a4b1d63)

**Comment by daromacarlo** [comment]
![Immagine 2024-01-19 135507](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/135441131/1ad865d5-fc6d-4262-9dce-4198fca8caef)
![Immagine 2024-01-19 135524](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/135441131/77c10f1a-4137-4e84-ab6f-c0dc8999a832)
![Immagine 2024-01-19 135540](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/135441131/5b72d04e-44c0-4e81-bf86-e34560123b02)
![Immagine 2024-01-19 135550](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/135441131/6726a916-29ee-4857-b5df-e3203ca469c5)
